pynndescent/utils.py

```diff
@@ -493,6 +494,7 @@ def new_build_candidates(
     new_candidate_neighbors = make_heap(n_vertices, max_candidates)
     old_candidate_neighbors = make_heap(n_vertices, max_candidates)
 
+    #    for i in numba.prange(n_vertices):
```

Comment:
`numba.prange` causes test failures (due to nondeterminism I think), and also doesn't confer any speed advantage when I tried it out, presumbly because numba can't do any meaningful parallelization of the loop.